doc/buildpack.md

```diff
@@ -1,9 +1,75 @@
 # Oryx Buildpack
 
-Oryx provides a buildpack that runs it, so that Oryx can also be used via the [pack][] tool.
+Oryx provides a buildpack that runs it, so that Oryx can also be used via the Cloud Native `[pack](https://github.com/buildpack/pack)` tool, with your own Azure Container Registry (ACR) using ACR Tasks with the Azure CLI `az acr pack` command.  
```

Comment:
keeping the links (like "[pack]") below helps when reviewing as well as reusing them.